chrome/locale/en-US/zotero/standalone.dtd

```diff
@@ -63,6 +63,7 @@
 <!ENTITY noteFontSize.label             "Note Font Size">
 <!ENTITY columns.label                  "Columns">
 <!ENTITY sortBy.label                   "Sort By">
+<!ENTITY moveColumn.label               "Move Column">
```

Comment:
"Back" is too confusing, since it has the "way it was before" sense too. So I think it'd have to be "Move Column Left" or "Move Column Right" depending on RTL mode.

This could also go above "Restore Column Order", in the same menu section.

Reply:
Yeah, that makes sense. Let's have "Move Column Left/Right" depending on the RTL mode. It sounds a tiny bit off to me, mainly because if you have "move something left", you somewhat expect to also have "move something right"? But it is a better alternative than using "Back" (which does have the Return/Undo meaning), for sure!

Reply:
I guess there's no real reason not to add "Move Column Right" as well? And the order of the menu options is determined by RTL.

Reply:
Initially, we had both "Move Column Right" and "Move Column Left" in this PR. But it was removed after some discussion above https://github.com/zotero/zotero/pull/4433#issuecomment-2272815151.

I agree that the menu space is generally quite valuable and we should only be using it for features that are important. From this perspective, having both "Move Column Right" and "Move Column Left" is a bit wasteful, since one can be achieved with the other. At the same time, it might be nice to have both for the sake of consistency. But I assume one is not going to be moving the columns in itemTree via a menu that often, so is consistency worth it in this case? These are the considerations, and both points seem valid to me.

I can bring back the "Move Column Right" to see what it would look like with the current setup

Reply:
This is pretty deep in a submenu, so I'm not really worried about menu space, but I guess it's true that this will be used pretty rarely, so a little awkwardness isn't that big a deal. Let's try it as is and see if it becomes an issue.